tutorials/preprocessing/plot_60_maxwell_filtering_sss.py

```diff
@@ -1,45 +1,197 @@
 """
 .. _tut-artifact-sss:
 
-=======================================
-Artifact correction with Maxwell filter
-=======================================
+Signal-space separation and Maxwell filtering
+=============================================
 
-This tutorial shows how to clean MEG data with Maxwell filtering.
+This tutorial covers reducing environmental noise and compensating for head
+movement with SSS and Maxwell filtering.
 
-Maxwell filtering in MNE can be used to suppress sources of external
-interference and compensate for subject head movements.
+.. contents:: Page contents
+   :local:
+   :depth: 2
 
-See :ref:`maxwell` for more details.
+As usual we'll start by importing the modules we need, loading some
+:ref:`example data <sample-dataset>`, and cropping it to save on memory:
 """
 
+import os
 import mne
-from mne.preprocessing import maxwell_filter
 
-data_path = mne.datasets.sample.data_path()
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file, verbose=False)
+raw.crop(tmax=60).load_data()
 
 ###############################################################################
-# Set parameters
-raw_fname = data_path + '/MEG/sample/sample_audvis_raw.fif'
-ctc_fname = data_path + '/SSS/ct_sparse_mgh.fif'
-fine_cal_fname = data_path + '/SSS/sss_cal_mgh.dat'
+# Signal-space separation (SSS) [1]_ [2]_ is a technique based on the physics
+# of electromagnetic fields. SSS separates the measured signal into components
+# attributable to sources *inside* the measurement volume of the sensor array
+# (the *internal components*), and components attributable to sources *outside*
+# the measurement volume (the *external components*). The internal and external
+# components are linearly independent, so it is possible to simply discard the
+# external components to reduce environmental noise. *Maxwell filtering* is a
+# related procedure that omits the higher-order components of the internal
+# subspace, which are dominated by sensor noise. Typically, Maxwell filtering
+# and SSS are performed together (in MNE-Python they are implemented together
+# in a single function).
```

Comment:
I actually didn't know there was a distinction (but there very well could be)! Is this adapted from the Maxfilter manual, or somewhere else?

Reply:
> This process of leaving out the high *l*, *m* end of the spectrum not only reduces the noise like ordinary spatial filtering but also guarantees that the retained field distribution is consistent with Maxwell's equations ∇ ⋅ **b** = 0 and the quasistatic ∇ ⨯ **b** = 0 in empty space. Therefore, we call this procedure "Maxwell filtering."

Taulu, S., Simola, J., & Kajola, M. (2005). Applications of the signal space separation method. IEEE Transactions on Signal Processing, 53(9), 3359–3372. https://doi.org/10.1109/TSP.2005.853302